### PR TITLE
[speech-commands] Add JavaScript training script for browser-FFT audio model (finally)

### PR DESCRIPTION
**Pull request by caisq**
Accompanying changes:
- Add `shuffle` and `normalize` options to `Dataset.getSpectrogramsAsTensors()`
- Add unit test for `normalize()`

This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/126)

### CODE REVIEW

**Comment by caisq** [comment]
cc @rusheywood for context. This PR is mainly about training a model from scratch on the original speech-commands dataset. So it isn't most relevant to transfer learning. But feel free to comment.
